benchmarks/src/jmh/kotlin/benchmarks/SemaphoreBenchmark.kt

```diff
@@ -83,7 +83,7 @@ enum class SemaphoreBenchDispatcherCreator(val create: (parallelism: Int) -> Cor
     EXPERIMENTAL({ parallelism -> ExperimentalCoroutineDispatcher(corePoolSize = parallelism, maxPoolSize = parallelism) })
 }
 
-private fun doWork(work: Int) {
+fun doWork(work: Int) {
```

Comment:
`SemaphoreBenchmark.kt` is not the best place for common functions. Let's create a special file for this.